nxt_cop.gemspec

```diff
@@ -1,4 +1,5 @@
 # coding: utf-8
```

Comment:
FYI: we can remove this magic comment 👀 . Ruby's default encoding has been UTF-8 for a long time.

Reply:
Good idea, thanks! 

Removed